src/main/java/com/github/kiulian/downloader/YoutubeDownloader.java

```diff
@@ -67,6 +68,14 @@ public Response<PlaylistInfo> getPlaylistInfo(RequestPlaylistInfo request) {
         return parser.parsePlaylist(request);
     }
 
+    public Response<SearchResult> search(RequestSearchResult request) {
+        return parser.parseSearchResult(request);
+    }
+
+    public Response<SearchResult> getNextPage(RequestSearchContinuation request) {
```

Comment:
I believe we need better naming for this method which indicates it is related to **search**, e.g. `searchNextPage` or `searchContinuation`

Reply:
Agree, bad naming.
However, it's not a "search" action (no search query, filter or sort order in there), I think it should not begin with "search".
Maybe `getNextSearchResult`, `getSearchContinuation` or something like this ?

Reply:
Looks like in `parseSearchContinuation` there is a http call to `https://www.youtube.com/youtubei/v1/search?key=`
so basically it is a search query, isnt it?
Anyway I am ok with `getSearchContinuation`, but in this case lets rename original `search` to `getSearchResult` to keep same style

Reply:
in my opinion, continuous scrolling == search action, even for user, it just the same as pagination - search for next page
maybe `getSearchContinuationResult` and `getSearchResult`?

Reply:
You convinced me, it's a search action with at least an extra parameter (offset).
If you don't mind, I'd like to get back to your first thought:
`search`
`searchContinuation`
I found another kind of result item: query refinement suggestion. So one more method:
`searchRefinement`